ui/packages/app/web/tailwind.config.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
       maxWidth: {
         '1/2': '50%',
       },
+      minWidth: theme => ({
+        ...theme('spacing'),
```

Comment:
Just curious, what does this do?

Reply:
@manojVivek this extends the tailwind css potential values available for setting 'minimum width' by making them the same as the options available for setting 'width'. Min-width had very few available options. For example, 'min-w-64' was not available, but 'w-64' (which equates to 16rem) was. So basically now, all the options available for width are also available for min width.

Reply:
Got it, thanks for the explanation, Monica. 
I wonder why wouldn't they do it by default. 🤔